js/actors/player.js

```diff
@@ -9,12 +9,13 @@ class Player extends Actor {
         this.controller = controller;
         this.grapple = null;
         this.animationFrameDuration = 20;
-        this.animationFrames = ['playerSprite']
+        this.animationFrames = ['playerSprite'];
         this.grappleStrengthX = 0.6;
         this.grappleStrengthY = 0.7;
         this.grappleLength = 650;
         this.accelerationCap = 1.5;
         this.extraPullStrength = 0.2;
+        this.arm = new Arm(this, this.controller);
```

Comment:
arm constructor doesnt take a controller reference: `constructor(player, x = 0, y = 300, w = 20 * 0.8, h = 36 * 0.8)`
also let's give all actors a reference to level so that arm doesnt have to call this.player.level